From 150ace55558cdb9fadc361db267974abe5a151b1 Mon Sep 17 00:00:00 2001
From: qianmenx <qianx.meng@intel.com>
Date: Thu, 9 May 2019 10:42:25 +0800
Subject: [PATCH 2330/2367] media: intel-ipu4: update metadata info for x3a

Message for Open Source:
update metadata info for x3a


Change-Id: Ic7555778372eff8e06af416a858c8aa0faf54655
Tracked-On: #PKT-2352
Signed-off-by: qianmenx <qianx.meng@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 drivers/media/i2c/crlmodule/crl_ox03a10_common.h | 11 +++++++++--
 1 file changed, 9 insertions(+), 2 deletions(-)

diff --git a/drivers/media/i2c/crlmodule/crl_ox03a10_common.h b/drivers/media/i2c/crlmodule/crl_ox03a10_common.h
index 5ef7e3df..9448930 100644
--- a/drivers/media/i2c/crlmodule/crl_ox03a10_common.h
+++ b/drivers/media/i2c/crlmodule/crl_ox03a10_common.h
@@ -530,8 +530,15 @@ static struct crl_register_write_rep ox03a10_1920_1088_12DCG[] = {
 	{ 0x460a, CRL_REG_LEN_08BIT, 0x0e },
 	/* embedded data */
 	{ 0x3208, CRL_REG_LEN_08BIT, 0x04 },
-	{ 0x3800, CRL_REG_LEN_08BIT, 0x03 },
-	{ 0x4800, CRL_REG_LEN_08BIT, 0x02 },
+	{ 0x483e, CRL_REG_LEN_08BIT, 0x02 },/* frame counter */
+	{ 0x3501, CRL_REG_LEN_08BIT, 0x02 },/* DCG exp */
+	{ 0x3581, CRL_REG_LEN_08BIT, 0x02 },/* VS exp */
+	{ 0x350a, CRL_REG_LEN_08BIT, 0x02 },/* HCG Dgain */
+	{ 0x354a, CRL_REG_LEN_08BIT, 0x02 },/* LCG Dgain */
+	{ 0x358a, CRL_REG_LEN_08BIT, 0x02 },/* VS Dgain */
+	{ 0x3508, CRL_REG_LEN_08BIT, 0x02 },/* HCG Again */
+	{ 0x3548, CRL_REG_LEN_08BIT, 0x02 },/* LCG Again */
+	{ 0x3588, CRL_REG_LEN_08BIT, 0x02 },/* VS Again */
 	{ 0x3208, CRL_REG_LEN_08BIT, 0x14 },
 	{ 0x3208, CRL_REG_LEN_08BIT, 0x05 },
 	{ 0x5000, CRL_REG_LEN_08BIT, 0x10 },
-- 
2.7.4

